finrl/env/env_stocktrading_cashpenalty.py

```diff
@@ -59,15 +59,15 @@ def __init__(
         sell_cost_pct=3e-3,
         date_col_name="date",
         hmax=10,
-        min_shares=1,
+        discrete_actions=False,
+        shares_increment=None,
```

Comment:
Let's default this to `1` so that it doesn't break if not being specified coincidentally with discrete_actions

Reply:
Sure